### PR TITLE
Fix artifact name

### PR DESCRIPTION
**Pull request by sudarshan0212**
Thank you for your contribution.
Before submitting this PR, please make sure:

- [ ] PR description and commit message should describe the changes done in this PR
- [ ] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [ ] Latest Code from master or specific release branch is merged to your branch
- [ ] No unwanted\commented\junk code is included
- [ ] No new warning upon build solution
- [ ] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [ ] Verify that changes are compatible with all relevant browsers and platforms.
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve all Codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Update the Help Library document to match any feature changes

## Summary by CodeRabbit

- **New Features**
  - Enhanced image handling functionality by allowing specification of width and height for static images.
  - Improved artifact file naming by including the artifact name along with the file name.

- **Updates**
  - Updated stylesheet links and script sources for the Ginger Web Client reports.
  - Modified artifact handling in reports to streamline data mapping and improve performance.

### CODE REVIEW

File: Ginger/Ginger/Actions/UserControls/UCArtifact.xaml.cs [hunk 1]
```diff
@@ -62,7 +62,7 @@ public string ArtifactFileName
                 string FileName = string.Empty;
                 if(!string.IsNullOrEmpty(ArtifactPath))
                 {
-                    FileName = System.IO.Path.GetFileName(ArtifactPath);
+                    FileName = ArtifactName + " (" + System.IO.Path.GetFileName(ArtifactPath) +")";
```
Comment (Maheshkale447) [inline Ginger/Ginger/Actions/UserControls/UCArtifact.xaml.cs:65]: Please use string interpolation

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]
Please also check codacy issues

File: Ginger/Ginger/UserControlsLib/ImageMakerLib/ImageMakerControl.xaml.cs [hunk 2]
```diff
@@ -748,13 +748,13 @@
...
-                    SetAsStaticImage("xml.png");
+                    SetAsStaticImage("xml.png" , null, 50);                    
                     break;
                 case eImageType.FileJSON:
-                    SetAsStaticImage("json.png");
+                    SetAsStaticImage("json.png", null, 50);
                     break;
                 case eImageType.FileJavascript:
-                    SetAsStaticImage("javascript.png");
+                    SetAsStaticImage("javascript.png", null, 50);
```
[the remark this answers is not in the document]
Comment (sudarshan0212) [reply Ginger/Ginger/UserControlsLib/ImageMakerLib/ImageMakerControl.xaml.cs:757]: The idea of default width=0 / Height=0 is to apply only if user specified the width / height explicilty. So here im checking if its default value and then ignoring it.

File: Ginger/Ginger/Actions/UserControls/UCArtifact.xaml.cs (hunk 1 above)
Comment (sudarshan0212) [reply Ginger/Ginger/Actions/UserControls/UCArtifact.xaml.cs]: Done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**